mungegithub/mungers/submit-queue.go

```diff
@@ -488,6 +499,9 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 		if sq.BatchEnabled {
 			http.Handle("/batch", gziphandler.GzipHandler(http.HandlerFunc(sq.serveBatch)))
 		}
+		if sq.ProwURL != "" {
+			http.Handle("/ci-status", gziphandler.GzipHandler(http.HandlerFunc(sq.serveCIStatus)))
```

Comment:
Good point. I only enabled it with prow because we obtain the status from prow. We could always enable it.